docs/src/SegmentedControl.doc.js

```diff
@@ -41,84 +40,69 @@ card(
         required: true,
         description: 'Index of element in `items` that is selected.',
       },
+      {
+        name: 'size',
+        type: '"md" | "lg"',
+        required: false,
+        description: 'Size of the Segemented control.',
+        defaultValue: 'md',
+      },
     ]}
     heading={false}
   />
 );
 
 card(
-  <Box
-    description={`
-    There are 2 different sizes for segmented controls. The default size is \`md\`.
-
-    ~~~html
-    <SegmentedControl
-      selectedItemIndex={0}
-      items={['Athos', 'Porthos', 'Aramis']}
-    />
-    ~~~
-
-    ~~~html
-    <SegmentedControl
-      selectedItemIndex={0}
-      size="lg"
-      items={['Athos', 'Porthos', 'Aramis']}
-    />
-    ~~~
-  `}
-    name="Sizes"
-  >
-    <Box padding={2}>
-      <SegmentedControl
-        onChange={() => {}}
-        selectedItemIndex={0}
-        items={['Athos', 'Porthos', 'Aramis']}
-      />
-    </Box>
-    <Box padding={2}>
-      <SegmentedControl
-        onChange={() => {}}
-        selectedItemIndex={0}
-        size="lg"
-        items={['Athos', 'Porthos', 'Aramis']}
-      />
-    </Box>
-  </Box>
-);
-
-card(
-  <StateRecorder
-    description={`
-    Segmented Controls are naive components, meaning you need to write up the behavior when you click on an item.
+  <Example
+    description="Segmented Controls are naive components, meaning you need to write up the behavior when you click on an item.
 
     If you'd like the tabs to control hiding or showing content that state should
     live in a parent component.
-
-    ~~~js
-    <SegmentedControl
-      selectedItemIndex={0}
-      items={['News', 'You', 'Messages']}
-      onChange={() => {}}
-    />
-    ~~~
-  `}
+    "
     name="Example"
-    fn={atom => {
-      const state = atom.deref();
-      return (
-        <SegmentedControl
-          items={['News', 'You', 'Messages']}
-          selectedItemIndex={0}
-          {...state}
-          onChange={({ activeIndex }) =>
-            atom.set(props => ({
-              ...props,
-              selectedItemIndex: activeIndex,
-            }))
-          }
-        />
-      );
-    }}
+    defaultCode={`
+    class ToastExample extends React.Component {
```

Comment:
@nithindavid Seems like the indentation in the docs looks a bit off because of this.

![image](https://user-images.githubusercontent.com/127199/37716005-b5da6a32-2cda-11e8-9be4-c74d10613532.png)